src/application/scene/fxml/Todo.fxml

```diff
@@ -1,11 +1,91 @@
+                                        </ImageView>
+                                    </graphic>
+                                </Button>
+                                <Button fx:id="button_moveDONE" layoutX="775.0" layoutY="263.0" mnemonicParsing="false" onAction="#moveToDONE" prefHeight="57.0" prefWidth="69.0" style="-fx-background-color: #6ec6ff;" AnchorPane.bottomAnchor="230.0" AnchorPane.leftAnchor="775.0" AnchorPane.rightAnchor="380.0" AnchorPane.topAnchor="290.0">
+                                    <graphic>
+                                        <ImageView fitHeight="30.0" fitWidth="30.0" pickOnBounds="true" preserveRatio="true">
+                                            <image>
+                                                <Image url="@../assets/icon_Todo_moveEvents.png" />
+                                            </image>
+                                        </ImageView>
+                                    </graphic>
+                                </Button>
+                                <Button fx:id="button_clearDONE" layoutX="1113.0" layoutY="61.0" mnemonicParsing="false" onAction="#clearDONE" style="-fx-background-color: #ffa4a2;">
+                                    <graphic>
+                                        <ImageView fitHeight="30.0" fitWidth="30.0" pickOnBounds="true" preserveRatio="true">
+                                            <image>
+                                                <Image url="@../assets/icon_Todo_removeDONE.png" />
+                                            </image>
+                                        </ImageView>
+                                    </graphic>
+                                </Button>
+                            </children>
+                        </AnchorPane>
+                        <AnchorPane layoutX="50.0" layoutY="14.0" minHeight="-Infinity" prefHeight="94.0" prefWidth="1210.0" style="-fx-background-color: #5d576e;" AnchorPane.leftAnchor="50.0" AnchorPane.rightAnchor="50.0" AnchorPane.topAnchor="14.0">
+                            <children>
+                                <TextField fx:id="textField_eventName" layoutX="47.0" layoutY="28.0" prefHeight="38.0" prefWidth="592.0" style="-fx-background-color: #e0e0e0;" styleClass="textField_eventName" />
+                                <DatePicker fx:id="datePicker" layoutX="672.0" layoutY="28.0" prefHeight="38.0" prefWidth="238.0" />
+                                <ChoiceBox fx:id="choiceBox_status" layoutX="933.0" layoutY="28.0" prefHeight="38.0" prefWidth="116.0" />
+                                <Button fx:id="button_addEvent" layoutX="1073.0" layoutY="26.0" mnemonicParsing="false" onAction="#addEvent" prefHeight="41.0" prefWidth="118.0" styleClass="button_addEvent" text="   Add" textFill="WHITE" />
+                                <ImageView fitHeight="30.0" fitWidth="30.0" layoutX="1083.0" layoutY="32.0" pickOnBounds="true" preserveRatio="true">
```

Comment:
The button is a little hard to spot. Could it be changed to purple so its more clear as a call to action?

Reply:
Fixed